common/models/entities_with_tags.go

```diff
@@ -0,0 +1,43 @@
+package models
+
+type EntitiesWithTags []*EntityWithTags
+
+type ewtFilter func(e EntityWithTags) bool
+
+func (e EntitiesWithTags) RemoveIf(f ewtFilter) EntitiesWithTags {
```

Comment:
To me, it just seems like the most logical place to put it. Also, I don't want to have to import 2 packages whenever I'd like to operate on tags.